login/url_dto.go

```diff
@@ -0,0 +1,13 @@
+package login
+
+type Url struct {
+	Name string `json:"url"`
+}
+
+type Urls struct {
+	Items []Url `json:"urls"`
+}
+
+func (urls *Urls) AddItem(item Url) {
```

Comment:
Can we change these lines to make return object like this:
```
    "URLs": [
             "hepsiburada.com",
             "abc.com",
             "cfg.com",
    ]
```

Reply:
ı think we may extend struct such as add favorite field, if you want again after you think , ı can do 

url | favorite
d1 | *
d3 | *
d2
d4
d6

Reply:
Thank you but I am really trying hard to keep it simple. Maybe in the future we should add extra features to compete with other companies but for now I think we don't need it.

Reply:
I can understand you :) I m doing :)